modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/package.scala

```diff
@@ -2,7 +2,37 @@ package com.twilio.guardrail.generators
 
 import cats.data.NonEmptyList
 import java.util.Locale
+import io.swagger.v3.oas.models.{ Operation, PathItem }
 import io.swagger.v3.oas.models.media.Schema
+import io.swagger.v3.oas.models.parameters.Parameter
+import scala.collection.IterableLike
+import scala.collection.generic.CanBuildFrom
+import scala.language.implicitConversions
+
+package syntax {
+  class RichNotNullShower[A](value: A) {
+    def showNotNull: String = showNotNullIndented(0)
+    def showNotNullIndented(indent: Int): String =
+      ("  " * indent) + value.toString().linesIterator.filterNot(_.contains(": null")).mkString("\n" + ("  " * indent))
+  }
+
+  class RichCollection[A, Repr](xs: IterableLike[A, Repr]) {
```

Comment:
from https://stackoverflow.com/a/3912833. `distinctBy` is a member of 2.13's `View`, so this can go away when we upgrade.